src/main/java/org/dcsa/ebl/controller/TransportDocumentController.java

```diff
@@ -61,13 +60,13 @@ public String getType() {
     }
 
     @GetMapping(path="{transportDocumentID}")
-    public Mono<TransportDocumentTO> findById(@PathVariable UUID transportDocumentID) {
+    public Mono<TransportDocumentTO> findById(@PathVariable String transportDocumentID) {
```

Comment:
This should be renamed to transportDocumentReferene
transportDocumentID has been removed and transportDocumentNumber has been renamed to transportDocumentReference and is to be used instead...